app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -215,12 +215,15 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
         throw new InternalError(InternalErrorReason.ServerError)
       }
 
-      // 425 code is returned by blockfrost API and indicates a full mempool
```

Comment:
I actually liked the blockfrost API reference comment being there :D Can you please put it back?

Reply:
yes, I thought that whoever wants to check it should just look up adalite-backend, but I agree putting it here explicitly is indeed more sensible